lib/tasks/fly.rake

```diff
@@ -27,7 +29,7 @@ namespace :fly do
   #    point where swap is rarely used.  'fly scale help' for details.
   #  - disable by removing dependency on the :server task, thus:
   #        task :server do
-  task :swapfile do
+  task swapfile: :environment do
```

Comment:
NIT: This line is part of 228c76b5318b7afee5432043a8590337225956be, but I'm not sure if it is related to asset recompilation. I'm not sure why the Rails environment would be a prerequisite for `swapfile` task. So just want to be sure this was not committed accidentally.

Reply:
Oh, I'm sure that rubocop auto correct it for me in commit hook XDD

Reply:
[fixed](https://github.com/qoosuperman/taverneer-backend/pull/17/commits/bdb11b42223c000b7ab4f2602fe2cc0c1ba7a1a2)